packages/components/src/EditableText/EditableText.scss

```diff
@@ -69,25 +73,42 @@ $tc-input-editable-text-input-weight: $form-base-font-weight !default;
 		}
 	}
 	& &-form {
+		width: 100%;
 		display: flex;
 		align-items: center;
-		padding-left: $padding-large;
+		padding: {
+			left: $padding-large;
+			right: $padding-large;
+		}
+
+		:global(.form-group) {
+			width: 100%;
+			margin: 0;
+		}
 
 		& .tc-editable-text-form-input {
 			@include input-text($tc-input-editable-text-input-weight);
-			width: 18rem;
+			width: 100%;
+			padding-right: 2 * $tc-circle-editable-text-size + $padding-large;
 			&::selection {
 				background: lighten($scooter, 25%);
 			}
 		}
 
 		&-buttons {
 			display: flex;
+			position: absolute;
+			right: $padding-large;
+			align-items: center;
+			justify-content: center;
+			height: 32px;
+			top: 0;
 
 			&-icon {
-				border: 0.2rem solid currentColor;
-				width: $tc-input-editable-text-size-large;
-				height: $tc-input-editable-text-size-large;
+				border: 0.1rem solid $dove-gray;
+				color: $dove-gray;
+				width: $tc-circle-editable-text-size;
+				height: $tc-circle-editable-text-size;
```

Comment:
the `margin-left: $padding-large;` seems to not be used anymore.
What could be done is replace it with the :last-child one `margin-left: $padding-small;` for the two here (and remove the one below)

Reply:
✅